content-src/asrouter/templates/ReturnToAMO/_ReturnToAMO.scss

```diff
@@ -0,0 +1,131 @@
+.ReturnToAMOOverlay,
+.amo + body.hide-main {  // sass-lint:disable-line no-qualifying-elements
+  background: $white;
```

Comment:
Not sure what colour exactly because there's not a specification in the invision, but this is supposed to be light grey (I'm assuming the same grey as the new tab page)